### PR TITLE
(#99) Dictionary definition but no search results

### LINKED ISSUE #99: For terms that have dictionary definition but no search results, the text "Resultados 1-0 de 0 para:" appears
### Issue description
*Description of the issue*
For terms that have dictionary definition but no search results, the text "Results 1-0 of 0" appears  instead of "Results 0–0 of 0" 

> **ESTIMATE** 20

### Steps to reproduce the issue

1.  Perform search with "xerogram"
2. See that below the dictionary definition it shows "Results 1-0 of 0" , though there is no result on the search results page

### What's the expected result?
-"Results 0–0 of 0"  should be displayed

### What's the actual result?
- "Results 1-0 of 0"  appears

### Additional details / screenshot

image

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by belaolsonNIH**
- terms that have a dictionary definition but no search results now show the text "Results 0-0 of 0"

Closes #99

### CODE REVIEW

**Review by sarinapadilla — APPROVED** [review]
This passes product review! @alinai can you take a look please?

**Comment by alinai** [comment]
@sarinapadilla , Getting the following page for the term "xenogram"
image

**Comment by alinai** [comment]
@sarinapadilla , verified with the term "xerogram", "azaspirane", "alefacept", "xeroradiograph"
Spanish: "acederilla","angioplastiac on globo", "azaspirano",ferumoxitol

Looks good to me.

File: src/components/molecules/search-results-list/search-results-list.jsx [hunk 1]
```diff
@@ -28,13 +28,17 @@ const SearchResultsList = ({
...
 	const positionInResults = (currentPage - 1) * resultsPerPage;
-	const fromPage = currentPage > 1 ? positionInResults + 1 : currentPage;
+	let fromPage = currentPage > 1 ? positionInResults + 1 : currentPage;
 	let toPage =
 		currentPage > 1 ? positionInResults + resultsPerPage : resultsPerPage;
 	if (toPage > results.totalResults) {
 		toPage = results.totalResults;
 	}
 
+	if (results.totalResults == 0) {
```
Comment (arcepaul) [inline src/components/molecules/search-results-list/search-results-list.jsx:38]: combine into an else if block examining the results.totalResults

**Review by arcepaul — APPROVED** [review]

**Pull request merged**